fiboa_cli/publish.py

```diff
@@ -0,0 +1,378 @@
+            return (
+                properties.get(reverse.get(name))
+                or properties.get(name)
+                or DESCRIPTIONS.get(name, "")
+            )
+
+        cols = [["Property", "**Data Type**", "Description"]] + [
+            [
+                s["name"],
+                re.search(r"\w+", s["type"])[0],
+                description(s["name"]),
+            ]
+            for s in stac_data["assets"]["data"]["table:columns"]
+            if s["name"] not in ("geometry", "bbox", "collection")
+        ]
+        widths = [max(len(c[i]) for c in cols) for i in range(3)]
+        aligned_cols = [[f" {c:<{w}} " for c, w in zip(row, widths)] for row in cols]
+        aligned_cols.insert(1, ["-" * (w + 2) for w in widths])
+        return "\n".join(["|" + "|".join(cols) + "|" for cols in aligned_cols])
+
+    def make_license(self):
+        text = ""
+        try:
+            data, properties = self.get_data_survey()
+            text = data["license"]
+            if getattr(self.converter, "license") not in (None, "", data["license"]):
+                text += "\n" + self.converter.license + "\n"
+
+            found = False
+            for _license in (data["license"], self.converter.license):
+                if not _license or "<(https://" in _license:
+                    continue
+
+                # Include full-license text
+                _license = _license.upper()
+                if _license in spdx_license_list.LICENSES:
+                    response = requests.get(
+                        f"https://raw.githubusercontent.com/spdx/license-list-data/refs/heads/main/text/{_license}.txt"
+                    )
+                    if response.ok:
+                        found = True
+                        text += f"\n\n{response.text}\n"
+                        break
+            if not found:
+                self.warning(f"License {text} could not be found in SPDX license list")
+
+        except Exception as e:
+            self.exception(e)
+        return text
+
+    def make_readme(self, file_name, stac):
+        version = Registry.get_version()
+        converter = self.converter
+        stac_data = json.load(open(stac))
+        count = stac_data["assets"]["data"]["table:row_count"]
+        data, properties = self.get_data_survey()
+        columns = self.readme_attribute_table(stac_data, properties)
+        urls = converter.get_urls() or "manually downloaded file"
+        urls = urls.keys() if isinstance(urls, dict) else [urls]
+        downloaded_urls = "\n".join([("  - " + url) for url in urls])
+
+        return f"""# Field boundaries for {converter.short_name}
+
+Provides {count} official field boundaries from {converter.short_name}.
+It has been converted to a fiboa GeoParquet file from data obtained from {data["provider"]}.
+
+- **Source Data Provider:** [{data["provider"]}]({data["homepage"]})
+- **Converted by:** {data["submitter"]}
```

Comment:
Is the submitter really always the person that converted it? Would doubt it for several cases, I think.

At this point in 99% of the cases the conversion was either done by you (most of the time) or me.

Maybe it makes more sense and gives some additional value to just write that it was converted by the fiboa project based on the data survey, and then link to the data survey? Then people can actually check for additional information and sources of the data.